builder/internals.go

```
@@ -90,6 +90,16 @@ func (b *Builder) commit(id string, autoCmd []string, comment string) error {
 			return err
 		}
 		defer container.Unmount()
+
+		// Make sure all resources are setup, but we can ignore errors
+		// Supress output from this
```

Just curious, why do we want to ignore errors here?